getting_started/first_2d_game/04.creating_the_enemy.rst

```diff
@@ -112,7 +111,7 @@ to the ``Mob`` and add this code:
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    func _on_visible_on_screen_notifier_2d_screen_exited():
+    func _on_VisibilityNotifier2D_screen_exited():
```

Comment:
This isn't really a comment on your PR, more a note for myself that I need to check why we're using a non automatically generated function name before this is merged.